_posts/2022-mm-dd-crypto-1/2022-mm-dd-crypto1-history-and-basics.markdown

```diff
@@ -0,0 +1,157 @@
+
+**Eve**
+Eve is the **e**vil attacker that tries to interject, manipulate, listen to, ... whatever it is Alice and Bob are discussing.
+
+**Oscar**
+Oscar is our **o**pponent and, similar to Eve, tries to interfere with the communication between Alice and Bob. He might occasionally show up for the sake of flavour.
+
+We may at times need to introduce additional parties. For the sake of simplicity, all evil attackers will use names that start with an E and all opponents will start with an O. All other actors will draw from the remainder of the alphabet.
+
+#### Notation
+When talking about cryptography, especially when doing to in theory, there is not really a way around using math one way or another. We will try to focus on "concept first". This means that we'll introduce math as necessary, explaining new symbols as they come into play instead of building a giant glossary in this post. 
+> Mathematical notation/definitions will be put in quotes.
+
+Also, we'll sometimes use examples to illustrate our points. If we do, we'll follow these guidelines:
+* `Cryptographic Information (plain/cipher texts, keys, ...) is represented by inline code`. This applies to entire sentences as well as single letters and words.
+* All examples will assume the standard latin alphabet of A to Z
+* We'll use uppercase letters only
+
+#### Glossary
+There's a few words you'll regularly come across when talking about cryptography. The following list will also only provide a first set of basics and will most probably be expanded as we go along.
+|word         |meaning                                                                                                          |
+|-------------|-----------------------------------------------------------------------------------------------------------------|
+|alphabet     |Set of characters (letters, numbers, symbols, ...) used to form texts. Need not be a conventional alphabet.      |
+|plain text   |Original message, unencrypted and readable by anyone.                                                            |
+|cipher (text)|Encrypted message, unintelligible and (hopefully) safe to pass on.                                               |
+|key (word)   |Phrase or word used to start the encryption algorithm.                                                           |
+|encrypting   |Using a cryptographic algorithm to convert plain text into a cipher.                                             |
+|decrypting   |Using a cryptographic algorithm to recover the original plain text from a cipher.                                |
+|attack       |Attempt to decrypt or modify a cipher text not intended for the attacker.                                        |
+|breaking     |Finding a way of quickly solving a cryptographic problem. May invalidate the security of an entire crypto system.|
+
+Now, with the preamble done, let's get to the matter at hand!
+
+### What are crypto systems?
```

Comment:
```suggestion
### What are cryptosystems?
```

Cryptosystem is commonly written as one word, even though English has no compositing like German.